src/php/Utente.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-require_once "./php/exceptions.php";
+require_once "../../exceptions.php";
```

Comment:
Un modo più pulito sarebbe fare così, in questo modo usiamo la posizione del file stesso come base piuttosto di quella del file che importa visto che potrebbe creare problemi.

```suggestion
require_once dirname(__FILE__) . "/exceptions.php";
```